templates/rhdm75-authoring-ha.yaml

```diff
@@ -173,12 +173,17 @@ parameters:
 - displayName: DataGrid Image
   description: DataGrid image.
   name: DATAGRID_IMAGE
-  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.1
+  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.2
+  required: true
+- displayName: DataGrid Container CPU Limit
+  description: DataGrid Container cpu limit.
+  name: DATAGRID_CPU_LIMIT
+  value: "4"
```

Comment:
I think this value is not correct, CPU limits on OCP are set in millicores.
I don't know hoe much cpu does jdg requires, but, I think something around 500m would be enough.
https://docs.openshift.com/container-platform/3.11/dev_guide/compute_resources.html#dev-compute-resources

Reply:
going with millicores